resources/scss/admin_chatroom.scss

```diff
@@ -0,0 +1,31 @@
+.avatar-wrap {
+    position: relative;
+    display: inline;
+}
+
+.avatar {
```

Comment:
you can add `.avatar` inside `.avatar-wrap` actually since in your markup, the image with class `avatar` is inside the `div` with class `avatar-wrap` so you can do it like
```
.avatar-wrap {
    .... other styles like the position and display
    
    .avatar {
       ...your avatar styles here
    }
}
```

Reply:
I changed it.